### PR TITLE
[DVCSMP-2680] Correctly configure Aeon Multisensor Gen5

### PR DESCRIPTION
**Pull request by varzac**
Previously this device had been changed to use configureAfterSecure to
work around the secure inclusion for Z-Wave.  However, support was added
hub side for handling this so it needed to be changed back to using the
normal configure process.

This resolves: https://smartthings.atlassian.net/browse/DVCSMP-2680

### CODE REVIEW

File: devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy [hunk 1]
```diff
@@ -152,7 +150,7 @@ def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulat
 
 def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityCommandsSupportedReport cmd) {
     // log.debug "Received SecurityCommandsSupportedReport"
-	response(configureAfterSecure())
+	response(configure())
```
Comment (varzac) [inline devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy]: Is this still necessary @mckeed ?

**Review by tpmanley — APPROVED** [review]

File: devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy (hunk 1 above)
Comment (mckeed) [reply devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy]: No, you should take out this entire handler method.

Comment (varzac) [reply devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy]: I suspected as much.

**Comment by varzac** [comment]
Pointed at staging so it can go out next week, but can switch back to master if we want to wait longer.  @workingmonk thoughts?

**Comment by workingmonk** [comment]
there isn't a ton for QA next week, so if you can talk to Steven about the testing, we can get it in. Do you want Duncan's review too?

**Comment by varzac** [comment]
Steven Said he could do it tomorrow so merging tomorrow morning would be good @mckeed could you take a look before then?

**Review by mckeed — APPROVED** [review]

**Comment by varzac** [comment]
@workingmonk @tpmanley This should be good merge when you get a chance.

**Pull request merged**